src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -989,6 +989,10 @@ private PSModuleInfo ImportModule_LocallyViaName(ImportModuleOptions importModul
                 try
                 {
                     this.ArgumentList = new object[] { psSession };
```

Comment:
I don't understand why the PSSession is added/reverted from the ArgumentList here.  The remote session has already been imported (via the Export-PSSession Cmdlet) and is no longer needed.  It looks like this can be removed.

Reply:
Actually this does need to be here.  It is used to fill the PSSession property on the module info.